hubmap/query_app/tests.py

```diff
@@ -108,19 +112,19 @@ def test_all_cells(self):
         all_cells_count = set_count(all_cells, "cell")
         self.assertEqual(all_cells_count, 30)
 
-    def test_cells_from_genes(self):
-        input_set = ["ABHD17A"]
-        gene_cells = hubmap_query(
-            input_type="gene", output_type="cell", input_set=input_set, genomic_modality="rna"
-        )
-        gene_cells_count = set_count(gene_cells, "cell")
-        self.assertEqual(gene_cells_count, 5)
+    #    def test_cells_from_genes(self):
```

Comment:
May as well delete these blocks rather than comment them out; they're present in the Git history and can be resurrected if desired. (Same comment applies to a few other occurrences.)